src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', resolve)
+  });
+
+
+const promise2 = new Promise((resolve, reject) => {
+    setTimeout(reject, 3000);
+  });
+
+const success = () => {
+  body.insertAdjacentHTML('beforeend', 
+  `<div class="message">Promise was resolved!</div>`
+  )
+}
+
+const error = () => {
+  body.insertAdjacentHTML('beforeend', 
+  `<div class="message error-message">Promise was rejected!</div>`
+  )
+}
+
+promise1.then(success, error);
+promise2.then(success, error);
\ No newline at end of file
```

Comment:
Add a newline at the end of the file for better readability and to follow best practices.
Also, better catch reject using catch() method
![image](https://user-images.githubusercontent.com/96409563/233459492-f4b44fd3-6d23-4586-a5f1-266cf54f4550.png)